src/backend/expungeservice/models/charge_types/charge.py

```diff
@@ -15,14 +14,14 @@ def __init__(self, case, name, statute, level, date, chapter, section, dispositi
         self.level = level
         self.date = datetime.date(datetime.strptime(date, '%m/%d/%Y'))
         self.disposition = disposition
-        type_eligibility = self.default_type_eligibility()
+        type_eligibility = self._default_type_eligibility()
         self.expungement_result = ExpungementResult(type_eligibility=type_eligibility, time_eligibility=None)
         self._chapter = chapter
         self._section = section
         self._case = weakref.ref(case)
 
-    def default_type_eligibility(self):
-        return TypeEligibility(EligibilityStatus.NEEDS_MORE_ANALYSIS, reason='Examine')
+    def _default_type_eligibility(self):
```

Comment:
Since this no longer has a default return value, could it be decorated with `@abstractmethod` for a tad of added clarity? Up to you though, since that'd definitely subjective.

Reply:
Thanks for the suggestion. I looked into adding the decorator and I don't feel its necessary here as the code is more clear now than what it was before. I trust developers will act accordingly when they see methods like this. Plus any attempt to initialize this class or a subclass without overriding the method will fail immediately. <- if that wasn't the case then I'd probably take you up on your suggestion.

I looked at it from this perspective. When someone see's this latch on the door then it conveys that it's meant to be pulled and that the door will swing open toward the user.
<img width="163" alt="Screenshot 2019-11-16 00 37 42" src="https://user-images.githubusercontent.com/19519317/68990609-fa94b900-0809-11ea-90fe-1f59d3c80152.png">

However, does the door really need a sign on it when the latch strongly suggests to pull at it?
<img width="165" alt="Screenshot 2019-11-16 00 33 35" src="https://user-images.githubusercontent.com/19519317/68990623-2879fd80-080a-11ea-93c8-0803770b4fcb.png">

Reply:
In rebuttal I shall cite the Zen of Python, line 2 :). (from `import this` , but everyone already knows that right?)

IMO `NotImplementedError` reads more like an implicit abstract method but could also be misinterpreted as a `TODO` line.